fastlane/lanes/release.rb

```diff
@@ -60,11 +60,14 @@
       set_upstream: is_ci == false # only set upstream when running locally, useless in transient CI builds
     )
 
+    # Copy the branch protection settings from the default branch to the new release branch
     copy_branch_protection(
       repository: GITHUB_REPO,
       from_branch: DEFAULT_BRANCH,
       to_branch: computed_release_branch_name
     )
+    # But allow admins to bypass restrictions, so that wpmobilebot can push to the release branch directly for beta version bumps
+    set_branch_protection(branch: computed_release_branch_name, enforce_admins: false)
```

Comment:
Do we need to also set a repository? 🤔 

```suggestion
    set_branch_protection(repository: GITHUB_REPO, branch: computed_release_branch_name, enforce_admins: false)
```

Reply:
Oof good catch! 👍